packages/Confirmation/src/Confirmation.js

```diff
@@ -54,6 +54,8 @@ const Confirmation = props => {
   const confirmId = React.useRef(uuid()).current;
   let popoverKey = uuid();
 
+  const popoverOffset = 4;
```

Comment:
Adding an offset that is the same as the `dropdownMenu` component

Reply:
Again, something we can aim to consolidate in a future pr with the dropdown component